client/packages/common/src/ui/forms/JsonForms/useJsonForms.tsx

```diff
@@ -179,17 +213,23 @@ export const useJsonForms = (
         paddingX={10}
       >
         <ScrollFix />
-        <FormComponent
-          data={data}
-          setData={updateData}
-          // setError={setError}
-          renderers={renderers}
-        />
-        {showButtonPanel && <ButtonPanel />}
+        {isLoading || !data ? (
+          <Pulse />
```

Comment:
I prefer the pulse.. but would still use the `<BasicSpinner />` because has the loading text